### ISSUE TITLE
Berücksichtigung von Mängeln bei Alternativversionen (z.B. Kontrastansicht)

### ISSUE THREAD

**Issue by detlevhfischer** [BIK-BITV/BIK-Web-Test#341]
Anlässlich einer Qualitätssicherung tauchte die Frage auf, wie Mängel, die bei Aktivierung eines Kontrastschalters entstehen, in die Bewertung eines Tests eingehen (sollten). Da das Thema sicherlich von generellem Interesse ist, beantworte ich eine Prüfer:in-Rückfrage hier mit diesem Issue.

In einer Qualitätssicherung hatte ich zurückgemeldet: 

> Grundsätzlich: Wären die Kontraste im Standardmodus ausreichend, könnte die "Kontrastversion" unberücksichtigt bei der normativen Bewertung bleiben. Wenn auch bei der Kontrastversion Mängel bestehen wie hier zumindest bei manchen Seiten, sollte sich die Bewertung dann auf Mängel der Standardversion beziehen (die Kontrastversion würde dann nicht als konforme Alternative für diese Seite akzeptiert, da sie selber Mängel aufweist). (...) Zusätzliche Hinweise zur Kontrastversion sind natürlich trotzdem gut.“
 
Daraufhin kam die Rückfrage:

> Wenn ich deinen Kommentar richtig verstanden habe, bedeutet das, dass die Zusatzoption explizit nur dann geprüft wird, wenn die Originalversion nicht überall 100% konform ist. Wenn auch die Zusatzversion Mängel aufweist, werden diese aber nicht bewertet, sondern ignoriert und nur die Mängel der Originalversion gewertet. Wenn die Originalversion konform ist, dann wird die Zusatzoption gar nicht betestet. Ist das so korrekt?
 
> In diesem Fall hat die kontrastreichere Version einige Mängel und macht den Kontrast teilweise schlechter als vorher, außerdem werden die Bilder gar nicht mehr angezeigt etc. Für den Nutzer, der ja erwartet, dass er hiermit eine Verbesserung erfährt, ist das doch sehr unangenehm. Haben wir hier gar keine Handhabe diese zusätzliche Option zu bewerten?

Der Mangel ist sicher real, und wenn solche Mängel bei Prüfen ins Auge fallen, sollten sie im besten Fall natürlich nicht ignoriert, sondern ggf. knapp vermerkt werden. Ob wir jedoch grundsätzlich fordern sollten, zusätzlich zur Bewertung, die sich immer auf **eine** der angebotenen Versionen bezieht - und im Fall von Mängeln bei der Alternativversion in der Regel auf die Standardversion - auch weitere Mängel aller angebotenen Versionen vollständig zu erfassen, wäre zu diskutieren. Dies würde den Prüfaufwand ggf. deutlich erhöhen und müsste sich letztlich im Preis des Tests niederschlagen. Natürlich sind zusätzliche Hinweise immer gut, wenn beim Durchsehen von Alternativversionen Dinge auffallen; aber aus Konformitätssicht müssen zusätzliche Versionen eben **nicht** alte Anforderungen erfüllen (und tun dies machmal bewusst nicht).

Weitere Stimmen zu diesem Thema sind willkommen!

**Comment by stefanFarnetani** [comment]
Ich stimme @detlevhfischer zu. Die WCAG fordert eine barrierefreie Version.
Um den Aufwand nicht unnötig zu steigern, ist es sinnvoll, nur eine Version zu testen.
Ich hatte irgendwo einmal gesehen, dass auf einer Website ein Styleswitcher umgesetzt war mit folgenden Versionen: 
- normal
- barrierefreie Version (da waren nur die Schriftfarben auf Schwarz gestellt)
- Hochkontrast-Version
- Dark Mode
In so einem Fall wäre es ausreichend, nur die "barrierefreie Version" genannte Version zu testen. Auch wenn aus ihrer Sicht dies die alternative Version ist.

Ich denke, eine gute Vorgehensweise ist:
Wenn in der Angebotsphase alternative Versionen (dazu würde ich auch Overlay zählen) entdeckt werden, muss der Kunden gefragt werden, wie er es gerne hätte.
Ist er mit den zusätzlichen Kosten einverstanden, kann die alternative Version mitgetestet werden.
